app/controllers/content_controller.rb

```diff
@@ -15,7 +15,7 @@ def api_request
   end
 
   def permitted_params
-    params.permit(:from, :to, :organisation_id, :document_type, :format, :page, :page_size, :date_range)
+    params.permit(:from, :to, :organisation_id, :document_type, :format, :page, :page_size, :date_range, :q)
```

Comment:
What's q?

Reply:
Query for searching? I thought it was quite an standard way... but I might be wrong?